mergify_engine/tests/functional/base.py

```diff
@@ -622,6 +625,18 @@ async def create_pr(
             await self.git("commit", "--no-edit", "-m", f"{title}, moved")
         await self.git("push", "--quiet", base_repo, branch)
 
+        if verified:
+            temporary_folder = tempfile.mkdtemp()
```

Comment:
The temporary folder must be deleted at the end of the test.

Reply:
If the test fail before this line the temporary folder will not be deleted. (See: https://docs.python.org/3/library/unittest.html#unittest.TestCase.addCleanup)